pipeline_dp/budget_accounting.py

```diff
@@ -1,20 +1,91 @@
 """Privacy budget accounting for DP pipelines."""
 
 
+class Budget:
+  """Manages the budget allocated for an operation."""
+
+  def __init__(self, weight, use_eps, use_delta, eps=None, delta=None):
+    """Constructs a Budget.
```

Comment:
No need to have eps/delta as arguments of the constructor (explanation: the prototype used eps/delta because it was an option of eager mode, let's for now go only with lazy mode, maybe we will not eager mode :) )